homeassistant/components/conversation/chat_log.py

```diff
@@ -38,21 +38,23 @@ def async_get_chat_log(
     user_input: ConversationInput | None = None,
 ) -> Generator[ChatLog]:
     """Return chat log for a specific chat session."""
-    if chat_log := current_chat_log.get():
-        # If a chat log is already active and it's the requested conversation ID,
-        # return that. We won't update the last updated time in this case.
-        if chat_log.conversation_id == session.conversation_id:
-            yield chat_log
-            return
+    # If a chat log is already active and it's the requested conversation ID,
+    # return that. We won't update the last updated time in this case.
+    if (
+        chat_log := current_chat_log.get()
+    ) and chat_log.conversation_id == session.conversation_id:
+        if user_input is not None:
```

Comment:
Should this be in a separate bug fix PR?

Reply:
https://github.com/home-assistant/core/pull/138173